Add Learn Architecture section and related pages. (#624)
AstarNetwork/astar-docs: docs/build/Introduction/astar_family.md

Swanky Node is a Substrate based blockchain configured to enable the smart contract module `pallet-contracts`, and other features that assist local development of Wasm smart contracts.
For more information about Swanky Node, check out the [Swanky Suite](https://docs.astar.network/docs/build/wasm/swanky-suite/) section.

### Zombienet

With Zombienet users can download arbitrary Relay Chain and parachain binaries (or use images) to setup a configurable local test network. Users will have access to all privileged actions on the Relay Chain and parachains, which simplifies testing. For more information about Zombienet, check out the [Build Environment](https://docs.astar.network/docs/build/environment/zombienet-testing) chapter.

### zkEVM Node

A local Validium instance based on Polygon CDK that includes the following components:

- zkEVM databases: data node, event, explorer L1 and L2, pool, state, and bridge service
- zkEVM node components: aggregator, approve service, sequencer and sequence sender, synchronizer
- L1 network (mock)
- Prover
- Explorers L1, L2
- JSON RPC explorer
- L2 gas pricer
- DAC: data availability service, dac setup committee
- zkEVM bridge service and UI

> For more information about zkEVM nodes including the rollup-based zKatana version, check out the [zkEVM Build section](/docs/build/zkEVM/zk-node/).

Reviewer: Please remove zkEVM part. There is no local node concept for L2.

To obtain test tokens from the faucet, please visit the Astar Portal and connect to Shibuya. If for any reason the faucet is empty, please contact the Astar team on Discord.

### zKatana (Sepolia)

An early iteration of the zkEVM testnet that used zk rollup architecture. Now deprecated.

### zKyoto (Sepolia)

zKyoto is a Validium-based network connected to the Ethereum Sepolia testnet and Polygon AggLayer. It is the testing and proving ground for dApps deployed on the Astar zkEVM.

## Mainnets

> Astar has three mainnets. One parachain on Kusama, One parachain on Polkadot, and a zkEVM-based Validium network on Ethereum Layer 2.

Reviewer: I suggest we change the ordering of the mainnets intro, to 1. Astar zkEVM 2. Astar Parachain 3. Shiden Parachain

### Shiden Parachain (Substrate)

Shiden is a parachain connected to the Kusama Relay Chain, and used to deploy and test new releases of Astar runtime in a live production (canary) environment. Shiden is not considered a testnet since SDN has its own tokenomics and value, but is used to validate and stabilize new software releases and upgrades for Astar Network.

The Shiden native token symbol is SDN.

### Astar Parachain (Substrate)

By now you may have guessed that Astar parachain is connected to Polkadot Relay chain.

The Astar native token symbol is ASTR.

### Astar zkEVM

> Astar zkEVM is a Layer 2 scaling solution for Ethereum, powered by Polygon AggLayer. Technically referred to as a Validium, the zkEVM combines zk rollups with modular data availability, synthesizing the best features of each.

Reviewer: ```suggestion
Astar zkEVM is an Ethereum Layer 2 network, powered by Polygon CDK. Technically referred to as Validium, it combines zk rollup technology with modular data availability, synthesizing the best features of each.
```
